runtests.py

```diff
@@ -8,18 +8,20 @@
 File for running tests programmatically.
 """
 
+# Standard library imports
+import sys
+
 # Third party imports
-import qtpy
 import pytest
 
 
```

Comment:
@jitseniesen, what if we move

    PYTEST_QT_API=pyqt4v2

to here (using `os.environ`, of course)?

That way our tests would run locally too. I mean, right now they only run correctly in Travis and Appveyor :-)